src/execution/riscv64/simulator-riscv64.h

```diff
@@ -125,11 +125,15 @@ inline int64_t mulhsu(int64_t a, uint64_t b) {
 }
 
 // Floating point helpers
+#define isBoxedF32(v) ((uint32_t)((v >> 32) + 1) == 0)
```

Comment:
to be consistent w/ `boxF32`, make `isBoxedF32` a static inline function as well.